sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -453,3 +455,131 @@ case class Murmur3Hash(children: Seq[Expression], seed: Int) extends Expression
     }
   }
 }
+
+/**
+ * A function that encrypts input using AES. Key lengths of 128, 192 or 256 bits can be used.
+ * 192 and 256 bits keys can be used if Java Cryptography Extension (JCE) Unlimited Strength
+ * Jurisdiction Policy Files are installed. If either argument is NULL or the key length is
+ * not one of the permitted values, the return value is NULL.
+ */
+@ExpressionDescription(
+  usage =
+    """_FUNC_(input, key) - Encrypts input using AES. Key lengths of 128, 192 or 256 bits can
+ be used. 192 and 256 bits keys can be used if Java Cryptography Extension (JCE) Unlimited Strength
+ Jurisdiction Policy Files are installed. If either argument is NULL or the key length is not one
+ of the permitted values, the return value is NULL.""",
+  extended = """> SELECT Base64(_FUNC_('ABC', '1234567890123456'));
```

Comment:
we should follow the style of other classes in this file, i.e. use double quotation marks and `\n`